dc_config/images/celery/startCeleryWorker

```diff
@@ -1,12 +1,12 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
 import os
 from subprocess import call
 
 for itm in os.environ.get("CELERY_SOURCE").split(','):
-    call(['pip','install',itm])
+    call(['pip3','install',itm])
 
 celery_queue=os.environ.get("CELERY_QUEUE","celery")
 log_level=os.environ.get("LOG_LEVEL","INFO")
 
-call(["celery", "-Q", celery_queue, "-l", log_level, "worker"])
+call(["su", "-p", "celery", "-c", "celery -Q {0} -l {1} worker".format(celery_queue,log_level)])
```

Comment:
No its not required , I was using it since the last time, I updated startCeleryWorker.
I have just checked out whether celery is kicking without "su" and its working so we can revert the code changed in the startCeleryWorker.

Reply:
Great! Revert it back.